pepper-apis/studybuilder-cli/studies/lms/study-events.conf

```diff
@@ -1590,6 +1590,26 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    #BLOOD KIT_CREATE event
+    {
+      "trigger": {
+        "type": "DSM_NOTIFICATION",
+        "dsmEvent": "SALIVA_SENT"
+      },
+      "action": {
+        "type": "CREATE_KIT",
+        "kitType": "BLOOD"
+      },
+      "preconditionExpr": """
+            (user.studies["cmi-lms"].forms["CONSENT"].hasInstance() && user.studies["cmi-lms"].forms["CONSENT"].questions["CONSENT_BLOOD"].answers.hasTrue())
+            || (user.studies["cmi-lms"].forms["CONSENT_ASSENT"].hasInstance() && user.studies["cmi-lms"].forms["CONSENT_ASSENT"].questions["CONSENT_ASSENT_BLOOD"].answers.hasTrue())
+            || (user.studies["cmi-lms"].forms["PARENTAL_CONSENT"].hasInstance() && user.studies["cmi-lms"].forms["PARENTAL_CONSENT"].questions["PARENTAL_CONSENT_BLOOD"].answers.hasTrue())
+            """
```

Comment:
Should we check that the consent is actually completed?

Reply:
updated in patch file to check if MEDICAL_RELEASE is completed which is after consent